packages/cozy-client/src/models/permission.js

```diff
@@ -139,3 +139,26 @@ export async function isDocumentReadOnly(args) {
     return undefined
   }
 }
+
+/**
+ * Returns if the shortcut for the sharing is created on the recipient's cozy
+ *
+ * To know if the shortcut is created on the recipient's cozy, we need to check
+ * the presence of the `instance` attribute on the sharing member. (since we
+ * are in a cozy to cozy sharing, we have only one member per "token")
+ *
+ * If this field is setted, it means the stack knows the url of the recipient's
+ * cozy. And if the stack knows that, we're sure that the stack has already created
+ * the shortcut on the recipient's cozy.
+ *
+ * @param {Array} included JSON API included fields. From getOwnPermissions response
+ */
+export const isShortcutCreatedOnTheRecipientCozy = included => {
```

Comment:
I wonder why it is "included" that is passed here instead of a shortcut document ?